src/Core.zig

```diff
@@ -21,6 +21,11 @@ options: structs.Options,
 /// (360.0 * core.delta_time)
 delta_time: f32 = 0,
 delta_time_ns: u64 = 0,
+
+/// The amount of time (in seconds) that has passed since the start of the application
+total_time: f32 = 0,
+total_time_ns: u64 = 0,
```

Comment:
Instead of exposing these two as fields here, I suggest users just call the function themselves for now. We don't know in practice how many people will use this API, and updating these fields every frame isn't free.